### DISCUSSION TITLE
Does this endpoint "https://api.openlogin.com/jwks" always return same jwks?

### DISCUSSION THREAD

**Discussion by bhagyamudgal** [Web3Auth/Web3Auth#948] [Help]
If it returns the same value every time, I can save the value as constant rather than making a new fetch request every time for server side verification.

**Comment by BboyStatix** [comment]
@bhagyamudgal we do rotate our keys periodically so it's better to call the endpoint.

**Reply by bhagyamudgal** [reply]
Thanks
